scripts/docker/build.sh

```diff
@@ -130,10 +134,18 @@ while [ "$1" != "" ]; do
                                    ;;
         astrobee_quick )           astrobee_quick="true"
                                    ;;
+        rolling_base )             rolling_base="true"
```

Comment:
The naming is a bit confusing here. One might think that `rolling_base` is a ROS2 version of the base Dockerfile such that you can build `astrobee` on top of it. But instead it's a base that you can build ROS2 on top of, like with `ros2_rolling_source.Dockerfile`, and then build `astrobee` on top of that.

I'm not sure what to suggest here exactly, but just noticing that as the number of targets in `build.sh` keeps expanding, it will get harder to understand how they're all supposed to relate to each other, and more apt names might help a bit.

Reply:
Given that this is just the ros image from source, I think we'll have to add 2 more which is astrobee_ros2_base and astrobee_ros2 (note that astrobee_base builds on top of the noetic docker image). Once we're set on only one rolling/ros2 install method, this can compacted into one Dockerfile, but for now having the alternative is nice because it gives me some flexibility.